server/src/main/java/org/opensearch/wlm/QueryGroupTask.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.wlm;
```

Comment:
@kaushalmahi12 @jainankitk should we rename this to workloadmanagement instead ?
I don't think using abbreviations for a package name is a good idea.

Reply:
I'm aware this was already approved in the previous PR - https://github.com/opensearch-project/OpenSearch/pull/14614

Reply:
We can take this up once all the feature related changes are merged since they are less priority at this point